flows/engine/engine.go

```diff
@@ -95,20 +97,6 @@ func ResumeFlow(env flows.FlowEnvironment, run flows.FlowRun, event flows.Event)
 	return run.Session(), nil
 }
 
-// initializes our context based on our flow and current context
-func initTranslations(run flows.FlowRun) {
-	// set our language based on our contact if we have one
-	contact := run.Contact()
-	if contact != nil {
```

Comment:
I'm not seeing where else we support runs with no contact so am assuming this isn't actually possible